examples/HelloWorld_publisher.cr

```diff
@@ -0,0 +1,9 @@
+require "amqp-client"
+
+AMQP::Client.start("amqps://user:password@hostname/vhost") do |c|
+  c.channel do |ch|
+    q = ch.queue("hello")
+    q.publish "Hello World!"
```

Comment:
`publish_confirm` is the safe method to use, makes sure that the server acknowledge that it recivied the message. It's very much slower when publshing thousnads per second, but i think we should promote the safety by default. 
```suggestion
    q.publish_confirm "Hello World!"
```